aws integration in ops_omgx, check ops_omgs/README.md for more info

petardenev opened the pull request:
adding AWS ECS Cloudformation integration done in bash, docker, python script for parsing docker-compose recipes and importing their environment variables to aws secrets manager, small go healthcheck service, secret2env small go implementation for gettihg secrets from aws secret manager

CAPtheorem reviewed (approved):
Still lots to do - but in the interest of getting this package into the monorepo, approve. The main documentation gaps relate to system restart and the details of the databases/secrets and other data that also need to be secured to ensure reliable disaster recovery.

InoMurko commented on ops_omgx/docker/go-ethereum/wait-for-l1.sh:
```diff
@@ -0,0 +1,84 @@
+#!/bin/bash
```
I'm a little bit confused what this service is.

Is this L1 Geth or Verifier?

Based on env vars below, it seems to be the verifier, but the path naming implies it's L1 Geth.

CAPtheorem replied:
The is the geth_L2 running in Sequencer mode. Right now, the optimism codes uses 5 different name for the L2, namely: l2geth, geth, geth_l2, go-ethereum, and l2-node. Assuming the top level folder naming is the definitive one, suggest we call this service the `l2geth`. So the correct name for this file/folder would be /docker/l2geth/ etc etc.

InoMurko reviewed (commented):
Do we plan to run the L1 geth node or are we going to use Infura?

petardenev commented:
Just added feature to ./cfn-devenv.sh to restart all services, e.g. first stop them all, then start all of them ..... also added --force, to make sure, if you do make changes in the secret and the new secrets added would require the removal of the /mnt/efs contents - this can also be done with the bash automation.
I've added answers to the README.md file, hope its clear : )
